lib/travis/api/v3/renderer/user_builds.rb

```diff
@@ -0,0 +1,6 @@
+module Travis::API::V3
+  class Renderer::UserBuilds < CollectionRenderer
+    type            :builds
+    collection_key  :builds
+  end
+end
```

Comment:
I pulled this directly from the renderer used on `/builds` thinking that its basically the same data (a collection of Builds) we're just using a different filter for finding those results.